PersistenceExtensionsTest unstable

This test failed in: https://ci.openhab.org/job/openHAB-Core/654/

```
[ERROR] Tests run: 18, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.358 s <<< FAILURE! - in org.openhab.core.persistence.extensions.PersistenceExtensionsTest
[ERROR] org.openhab.core.persistence.extensions.PersistenceExtensionsTest.testHistoricSwitchState  Time elapsed: 0.004 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <null> but was: <org.openhab.core.persistence.extensions.TestPersistenceService$1@50029372>
        at org.openhab.core.persistence.extensions.PersistenceExtensionsTest.testHistoricSwitchState(PersistenceExtensionsTest.java:120)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   PersistenceExtensionsTest.testHistoricSwitchState:120 expected: <null> but was: <org.openhab.core.persistence.extensions.TestPersistenceService$1@50029372>
[INFO]
[ERROR] Tests run: 18, Failures: 1, Errors: 0, Skipped: 0

```

--- a/bundles/org.openhab.core.persistence/src/test/java/org/openhab/core/persistence/extensions/PersistenceExtensionsTest.java
+++ b/bundles/org.openhab.core.persistence/src/test/java/org/openhab/core/persistence/extensions/PersistenceExtensionsTest.java
@@ -169,8 +169,8 @@ public class PersistenceExtensionsTest {
 
     @Test
     public void testHistoricSwitchState() {
-        ZonedDateTime now = ZonedDateTime.now().truncatedTo(ChronoUnit.HOURS);
-        HistoricItem historicItem = PersistenceExtensions.historicState(switchItem, now.minusHours(15).minusMinutes(1),
+        ZonedDateTime now = ZonedDateTime.now().truncatedTo(ChronoUnit.HOURS).minusMinutes(1);
+        HistoricItem historicItem = PersistenceExtensions.historicState(switchItem, now.minusHours(15),
                 TestPersistenceService.ID);
         assertNull(historicItem);
 

--- a/bundles/org.openhab.core.persistence/src/test/java/org/openhab/core/persistence/extensions/PersistenceExtensionsTest.java
+++ b/bundles/org.openhab.core.persistence/src/test/java/org/openhab/core/persistence/extensions/PersistenceExtensionsTest.java
@@ -170,7 +170,7 @@ public class PersistenceExtensionsTest {
     @Test
     public void testHistoricSwitchState() {
         ZonedDateTime now = ZonedDateTime.now().truncatedTo(ChronoUnit.HOURS);
-        HistoricItem historicItem = PersistenceExtensions.historicState(switchItem, now.minusHours(15),
+        HistoricItem historicItem = PersistenceExtensions.historicState(switchItem, now.minusHours(15).minusMinutes(1),
                 TestPersistenceService.ID);
         assertNull(historicItem);
 

